htmlmaker_preprocessing.ps1

```diff
@@ -32,35 +35,45 @@ Function LogWrite
    Add-content $Logfile -value "$logstring"
 }
 
-# if original file name is a .doc file
-$fileType = ".doc"
-If ($filenameSplit -eq $filename + $fileType)
-{
-	# Converts a word .doc in bookmaker_tmp to a .docx
-	# so you have to have already run tmparchive.rb
-	write-host "Converting $filenameSplit to .docx from $fileType"
 
-	$SaveFormat = "microsoft.office.interop.word.WdSaveFormat" -as [type]
-	$word = New-Object -ComObject word.application
-	$word.visible = $false
+# Converts a word .doc in bookmaker_tmp to a .docx
+# so you have to have already run tmparchive.rb
+write-host "Converting $filenameSplit to .docx from $fileType"
+
+$SaveFormat = "microsoft.office.interop.word.WdSaveFormat" -as [type]
+$word = New-Object -ComObject word.application
+$word.visible = $false
+
+# Make sure "Store random number..." setting is off 
+$word.Options.StoreRSIDOnSave = $false
+
+$doc = $word.documents.open($docpath + $fileType)
 
-	$doc = $word.documents.open($docpath + $fileType)
-	$wdFormatDocx = 16  # wdFormatDocumentDefault is docx, reference number is 16
+# turn off track changes, accept all changes and delete all comments
+# have to do here, or we don't solve dumb XML revision problem
+$doc.TrackRevisions = $false
+$doc.Revisions.AcceptAll
```

Comment:
If it display a "No tracked changes" notification on a file without tracked changes showing, can use `$word.displayalerts = $false` and if it throws an error, can check for changes with `If ($doc.Revisions.Count -gt 1) { ... }`